lib/mincer/compressors/csswring_compressor.js

```diff
@@ -43,15 +48,25 @@ require('util').inherits(CsswringCompressor, Template);
 CsswringCompressor.prototype.evaluate = function (context/*, locals*/) {
   var result;
 
+  var wring = function (css, config) {
+    if (postcss && postcss.plugin &&
+        csswring.postcss && csswring.postcss.postcssPlugin) {
+      // Use newer API for csswring if it supports the Postcss plugin API
+      return postcss([ csswring(config) ]).process(css);
+    }
+
+    return csswring.wring(css, config);
+  };
+
```

Comment:
This should go to constructor - it needs to be initialized only once not upon each file evaluation.